aeneas/src/main/Version.v3

```diff
@@ -3,6 +3,6 @@
 
 // Updated by VCS scripts. DO NOT EDIT.
 component Version {
-	def version: string = "III-6.1492";
+	def version: string = "III-6.1494";
```

Comment:
Can you rebase? I've changed a few things today.

Reply:
No problem!

Reply:
Thanks, can you also bump the version number?

Reply:
Bumped!